fastlane/Fastfile

```diff
@@ -383,6 +377,34 @@ platform :android do
     )
   end
 
+  desc 'Update config and module/listener for Android telemetry'
+  lane :configure_telemetry_android do
+    if ENV['BETA_BUILD'] == 'true' && ENV['TELEMETRY_ENABLED'] == 'true' && ENV['TELEMETRY_URL'] != '' && ENV['TELEMETRY_API_KEY'] != ''
```

Comment:
By removing the check on `BETA_BUILD` we can only include telemetry if the others are set and that could help if we deploy this later on to production without any code changes.

Reply:
I agree, now updated. Thanks